utils/data-migration/export-tables.js

```diff
@@ -100,8 +100,9 @@ const exportAllCollections = async (collections) => {
 
       obj.files.push(table);
       // write the file to the ./exports directory
-      fs.writeFile(`${exportDirectory}/${date}/${table}.json`, JSON.stringify(result, null, 2), (error) => {
-        if (error) throw error;
+      // Defined `e` since `error` is defined on a higher scope
+      fs.writeFile(`${exportDirectory}/${date}/${table}.json`, JSON.stringify(result, null, 2), (e) => {
```

Comment:
probably more readable to have it defined as writeFileError rather than e

Reply:
Have renamed.

Reply:
Have updated. Thanks